src/Symfony/Component/PropertyInfo/Tests/Extractor/ReflectionExtractorTest.php

```diff
@@ -233,7 +233,7 @@ public function php71TypesProvider()
     }
 
     /**
-     *     * @dataProvider php80TypesProvider
```

Comment:
should be fixed upstream? right?

Reply:
I think it's fine to fix it in this PR. I'm targeting our lowest branch already.